src/core/utils/helpers/general.ts

```diff
@@ -562,12 +562,89 @@ export const getContributors = (short: boolean, creators: string[]) => {
   return creators[0];
 };
 
+export const getReservableOnAnotherLibrary = (
+  manifestations: Manifestation[]
+) => {
+  const matchingManifestations = manifestations.filter(({ catalogueCodes }) =>
+    catalogueCodes?.otherCatalogues.some((code) => code.startsWith("OVE"))
+  );
+
+  return {
+    isReservable: matchingManifestations.length > 0,
+    pids: matchingManifestations.map(({ pid }) => pid)
```

Comment:
does it make sense to return two values here, when they are both dealing with `matchingManifestations`?
If you call the function:
```typescript
getReservablePidsFromAnotherLibrary()
```
...you could just return the pids.
And then you know that if there are pids coming from the function, then there is reservable material from another library.